docs/src/app/components/DatePicker/ExampleDatePickers.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import DatePicker from 'material-ui/lib/date-picker/date-picker';
+import TextField from 'material-ui/lib/text-field';
+import Toggle from 'material-ui/lib/toggle';
+
+
+const DatePickerExamples = React.createClass({
```

Comment:
The name is not consistent with `docs/src/app/components/DatePicker/ExampleDatePickers.jsx`
Can we call this class `DatePickerExampleSimple` and the file `ExampleSimple.jsx`?